database/migrations/2023_02_19_035029_add_parent_template_id.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class () extends Migration {
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::table('email_templates', function (Blueprint $table) {
```

Comment:
why email_templates and not just "templates" ? 
In the future I can use that for generate files as XML

Reply:
@FredPeal we will change it to email, but for now we keep the legacy name